bin/critterget.py

```diff
@@ -164,6 +174,181 @@ def apicall_with_response_code(uri, attribs=None):
         return response.status_code, None
 
 
+def time_floor(last_run_time):
```

Comment:
It is a solution to the problem of figuring out when to make hourly and daily calls -- see my answer below. I am open to other solutions.